runtimes/aleph-alpine-3.13-python/init1.py

```diff
@@ -325,12 +325,22 @@ async def make_request(session, scope):
     return headers, body
 
 
+def show_loading():
+    f = open("/root/loading.html", "r")
```

Comment:
Use a context manager to ensure the file will be closed.
`with open(...) as f: ...`